Dockerfile

```diff
@@ -24,21 +36,36 @@ ADD . /usr/local/src/htm.core
 WORKDIR /usr/local/src/htm.core
 
 # Setup py env
-#RUN python3 -m venv pyenv && . pyenv/bin/activate
-RUN pip3 install --upgrade setuptools pip wheel
-#RUN export PYTHONPATH=$PYTHONPATH:/usr/local/lib/python3.7/dist-packages
+#! RUN python3 -m venv pyenv && . pyenv/bin/activate && python --version
+
+RUN ln -s /usr/bin/python3 /usr/local/bin/python && python --version 
+
+RUN python -m pip install --upgrade setuptools pip wheel
 
 # Install
-RUN pip3 install \
+RUN pip uninstall -y htm.core
+RUN python -m pip install \
 # Explicitly specify --cache-dir, --build, and --no-clean so that build
 # artifacts may be extracted from the container later.  Final built python
 # packages can be found in /usr/local/src/htm.core/bindings/py/dist
 #        --cache-dir /usr/local/src/htm.core/pip-cache \
 #        --build /usr/local/src/htm.core/pip-build \
 #        --no-clean \
         -r requirements.txt
-RUN python3 setup.py install --force
+RUN python setup.py install --force
 
 # Test
-RUN python3 setup.py test
+RUN python setup.py test #Note, if you get weird import errors here, 
+# do `git clean -xdf` in your host system, and rerun the docker
+
+## Stage 2: create release packages (for PyPI, GH Releases)
```

Comment:
Do users need to know how to build wheels?
I guess it is a good thing to document how to do it for us maintainers.

Reply:
> Do users need to know how to build wheels? 

I don't think so (?) 
And we also do it the standard way, `python setup.py bdist_wheel`